src/runtime_src/tools/scripts/xrtdeps.sh

```diff
@@ -99,9 +98,17 @@ rh_package_list()
          systemd-devel \
          python3 \
          python3-pip \
-         systemd-devel \
```

Comment:
Did you try building XRT from a clean centos8?  I am pretty sure this list is needed as it was.

Reply:
@stsoe Looks like these are duplicates.

Reply:
Duh :-)